archinstall/default_profiles/desktops/plasma.py

```diff
@@ -12,12 +12,16 @@ def __init__(self) -> None:
 	@override
 	def packages(self) -> list[str]:
 		return [
-			"plasma-meta",
-			"konsole",
-			"kate",
-			"dolphin",
 			"ark",
-			"plasma-workspace"
+			"bluedevil",
+			"breeze-gtk",
```

Comment:
- `breeze-gtk` nice to have for theme, not essential
- `plasma-nm` may not be wanted if the users did not setup any networking in the settings 
- `plasma-pa` also depends on the configuration of the audio server installed and may not be wanted
- `sddm-kcm` this should probably only be installed if SDDM was choosen as the display manager